.github/workflows/pull-request.yaml

```diff
@@ -88,7 +88,7 @@ jobs:
           cache: true
           cache-dependency-path: go.work
       - name: Run unit tests
-        run: cd server; make test
+        run: cd server; make test -B
```

Comment:
For some reason, this was not running the tests and was printing `test is up-to-date`. Adding the `-B` forces make to run this regardless of the cache.